src/OnboardingSPA/store/reducer.js

```diff
@@ -35,6 +35,7 @@ export function flow(
 		},
 		chapter: undefined,
 		interactionDisabled: false,
+		sitegenThemeMode: '',
```

Comment:
I doubt if this is required as we always have the data in currentData.sitegenThemeMode, then why here?